parser.go

```diff
@@ -281,6 +286,38 @@ func (p *parser) CreateOASFile(path string) error {
 	return err
 }
 
+func (p *parser) validateSchemaNames() []string {
+	potentialConflictsMap := map[string][]string{}
+	for pkgName, schemaNames := range p.ApiSchemaNames {
+		for typeName, schemaName := range schemaNames {
+			recordPotentialConflict(pkgName, typeName, schemaName, potentialConflictsMap)
+		}
+	}
+	conflicts := []string{}
+	for schemaName := range potentialConflictsMap {
+		if len(potentialConflictsMap[schemaName]) > 1 {
+			conflicts = append(conflicts, schemaName+": "+strings.Join(potentialConflictsMap[schemaName], " | "))
+		}
+	}
+	return conflicts
+}
+
+func recordPotentialConflict(pkgName, typeName, schemaName string, potentialConflictsMap map[string][]string) {
+	conflictName := pkgName + "#" + typeName
+	if potentialConflicts, ok := potentialConflictsMap[schemaName]; ok {
+		for i, potentialConflict := range potentialConflicts {
+			if strings.HasSuffix(potentialConflict, "/vendor/"+conflictName) {
+				// replace with shorter version
+				potentialConflictsMap[schemaName][i] = conflictName
+				return
+			} else if strings.HasSuffix(conflictName, "/vendor/"+potentialConflict) {
+				return
```

Comment:
Here we're just ignoring any types in the `vendor` dir that's a potential conflict only?

Reply:
Oh, this isn't necessary anymore, will remove 👍